src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -120,6 +120,25 @@ public function iAddThisProductToTheCart(ProductInterface $product, ?string $tok
         $this->sharedStorage->set('product', $product);
     }
 
+    /**
+     * @When /^I add ("[^"]+" variant) of (product "[^"]+") to the (cart)$/
```

Comment:
You can add this step to `iAddVariantOfThisProductToTheCart` method as the product is not needed here or use private method `putProductVariantToCart` instead of this implementation